services/payload/src/components/SideNav/SideNav.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import './SideNav.scss';
+import { Link, NavLink } from 'react-router-dom';
+import { useConfig } from 'payload/dist/admin/components/utilities/Config';
+import Logout from '../Logout/Logout';
+import Account from '../Account/Account';
+import { useAuth } from 'payload/dist/admin/components/utilities/Auth';
+
+const SideNav: React.FC = () => {
+    const { user } = useAuth();
+
+    const {
+        routes: {
+          admin,
+        },
+      } = useConfig();
+
+  return (
+    <div className="navbar-wrapper">
+        <div className="navbar-icon-wrapper">
+            <img src="/assets/tdm-alt-logo.png" alt="Tec de Monterray logo"/>
+        </div>
+        <div className="navbar-buttons-wrap">
+            <NavLink className="navbar-buttons" to="/admin/collections/credential-batch">
+                <img src="/assets/list-checks.svg" alt="credential-batch"/> Issuance Overview
+            </NavLink>
+            <NavLink className="navbar-buttons" to="/admin/collections/credential">
+                <img src="/assets/file-check.svg" alt="credential"/> Credentials
+            </NavLink>
+            <NavLink className="navbar-buttons" to="/admin/collections/credential-template">
+                <img src="/assets/file-edit.svg" alt="credential-template"/> Credential Templates
+            </NavLink>
+            <NavLink className="navbar-buttons" to="/admin/collections/email-template">
+                <img src="/assets/mail-plus.svg" alt="email-template"/> Email Templates
```

Comment:
This is totally fine for `src`, especially since these are SVGs anyways, but I usually prefer to import the file directly and then use that as `src`, because (depending on the build setup) it will usually optimize those images! Also, for SVGs I usually like to make an actual component for them because then it makes it easier to change their color!